example/app/data/DemoDataSource/DemoPackage/Plugins/Form/SingeField.json

```
@@ -0,0 +1,18 @@
+{
+  "name": "SingleField",
+  "type": "dmss://system/SIMOS/Blueprint",
+  "attributes": [
+    {
+      "name": "foo",
+      "type": "system/SIMOS/BlueprintAttribute",
```

```suggestion
      "type": "dmss://system/SIMOS/BlueprintAttribute",
```

same for all other types in the file